treeherder/config/settings.py

```diff
@@ -505,53 +505,42 @@ def server_supports_tls(url):
 PULSE_DATA_INGESTION_SOURCES = env.json(
     "PULSE_DATA_INGESTION_SOURCES",
     default=[
-        # {
-        #     "name": "exchange/taskcluster-treeherder/v1/jobs",
-        #     "projects": [
-        #         'mozilla-central',
-        #         'mozilla-inbound'
-        #         # other repos TC can submit to
-        #     ],
-        #     "destinations": [
-        #         'production'
-        #         'staging'
-        #     ]
-        # },
-        # {
-        #     "name": "exchange/treeherder-test/jobs",
-        #     "projects": [
-        #         'mozilla-inbound'
-        #     ],
-        #     "destinations": [
-        #         'production'
-        #         'staging'
-        #     ]
-        #
-        # }
+        {
+            "exchange": "exchange/taskcluster-treeherder/v1/jobs",
+            "projects": [
+                'mozilla-central.#',
+                'mozilla-inbound.#'
+                # other repos TC can submit to
+            ],
+            "destinations": [
+                'production'
+                # 'staging',
+            ]
```

Comment:
I set only one destination, `production`, if that makes sense. I left staging commented out.

Reply:
So here's the requested tweak. I figured leaving in the other separate examples made sense, and I adjusted the comment wording, since this is an ingestion variable?